Dockerfile

```diff
@@ -1,15 +1,33 @@
-FROM node:6.11-alpine
+FROM node:8.2.1-alpine
 
 # Create app directory
 RUN mkdir -p /usr/src/app
+RUN mkdir -p /usr/src/app/client
+RUN mkdir -p /usr/src/app/server
 WORKDIR /usr/src/app
 
 # Install app dependencies
 COPY package.json /usr/src/app/
+COPY package-lock.json /usr/src/app/
+COPY yarn.lock /usr/src/app/
+
+COPY server/yarn.lock /usr/src/app/server
+COPY server/package.json /usr/src/app/server
+COPY server/package-lock.json /usr/src/app/server
+
+COPY client/yarn.lock /usr/src/app/client
+COPY client/package.json /usr/src/app/client
+COPY client/package-lock.json /usr/src/app/client
+
 RUN npm install yarn -g
-RUN npm install
+RUN yarn
 
 # Bundle app source
 COPY . /usr/src/app
 
+# Build client assets
+WORKDIR /usr/src/app/client
+RUN yarn build

```

Comment:
Build is failing

See attached output
[TerminalOutput.txt](https://github.com/CatalystCode/ibex-dashboard/files/1189490/TerminalOutput.txt)